roles/mediawiki/files/lfc-analysis-rule.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Some comment somewhere near the top of this file, explaining what it's about / what it's for, would be good.  Most people won't see "lfc-analysis" and automatically associate that with "pages that are conditionally visible and may have UI changes designed to make them behave differently from main proposal pages".